reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/generation/code/generator/Util.kt

```diff
@@ -17,8 +17,6 @@ private fun FunSpec.Builder.generateBody(
     return build()
 }
 
-fun statement(format: String, args: List<Any>): CodeBlock = statement(format, *args.toTypedArray())
```

Comment:
I moved this method to the place with it's only usage. It should be better for a performance.